test/common/router/router_test.cc

```diff
@@ -213,7 +213,15 @@ TEST_F(RouterTest, MaintenanceMode) {
 
   Http::TestHeaderMapImpl headers;
   HttpTestUtility::addDefaultHeaders(headers);
+  EXPECT_EQ(
```

Comment:
I think @RomanDzhabarov meant to put this on a different test case. I would put it on a case where a 503 is responded with that is not maintenance mode.